src/components/main-view/main-view.jsx

```diff
@@ -1,37 +1,37 @@
-import React, { useState } from 'react'; 
+import React, { useState } from "react";
 import { MovieView } from "../movie-view/movie-view.jsx";
-import { MovieCard } from "../movie-card/movie-card.jsx"; 
+import { MovieCard } from "../movie-card/movie-card.jsx";
 import { useState, useEffect } from "react";
 import { LoginView } from "../login-view/login-view.jsx";
 import { SignupView } from "../signup-view/signup-view.jsx";
-
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import '../movie-card/movie-card.css';
 
 export const MainView = () => {
-  const [movies, setMovies] = useState([]); 
-  
+  const [movies, setMovies] = useState([]);
+
   const [selectedMovie, setSelectedMovie] = useState(null);
 
   const storedUser = JSON.parse(localStorage.getItem("user"));
-  const storedToken = localStorage.getItem("token"); 
+  const storedToken = localStorage.getItem("token");
 
-  const [user, setUser] = useState(storedUser? storedUser : null);
+  const [user, setUser] = useState(storedUser ? storedUser : null);
 
-  const [token, setToken] = useState(storedToken? storedToken : null); 
-  
- 
+  const [token, setToken] = useState(storedToken ? storedToken : null);
 
   useEffect(() => {
     if (!token) {
       return;
     }
     fetch("https://flixhive-cf7fbbd939d2.herokuapp.com/movies", {
-      headers: {Authorization: 'Bearer ${token}'},
+      headers: { Authorization: "Bearer ${token}" },
```

Comment:
@cinchronicity , Kindly revert the change on this line. Your movies API needs to be a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)(written with backticks NOT double quotes). 

```suggestion
      headers: { Authorization: `Bearer ${token}`},
```